gitlab-pipeline/stage/build.yml

```diff
@@ -26,27 +25,21 @@ build:client:docker:
   before_script:
     # Default value, will later be overwritten if successful
     - echo "failure" > /JOB_RESULT.txt
+    # Check out integration repository
+    - git init ${CI_PROJECT_DIR}/integration
+    - cd ${CI_PROJECT_DIR}/integration
+    - git remote add origin https://github.com/mendersoftware/integration.git
+    - git fetch --update-head-ok --force origin $INTEGRATION_REV:pr
+    - git checkout pr || git checkout --force -b pr $INTEGRATION_REV
     # Dependencies
     - apk --update add python3 py-pip curl jq bash git xz
     - wget https://raw.githubusercontent.com/mendersoftware/integration/master/extra/requirements.txt
     - pip3 install --break-system-packages -r requirements.txt
     # Post job status
     - ${CI_PROJECT_DIR}/scripts/github_pull_request_status pending "Gitlab ${CI_JOB_NAME} started" "${CI_JOB_URL}" "${CI_JOB_NAME}/${INTEGRATION_REV}"
-    # Prepare workspace
-    - export WORKSPACE=$(realpath ${CI_PROJECT_DIR}/..)
-    - mv workspace.tar.xz /tmp
-    - rm -rf ${WORKSPACE}
-    - mkdir -p ${WORKSPACE}
-    - cd ${WORKSPACE}
-    - xz -d /tmp/workspace.tar.xz
-    - tar -xf /tmp/workspace.tar
   script:
     - echo ${CI_REGISTRY_PASSWORD} | docker login --username ${CI_REGISTRY_USER} --password-stdin ${CI_REGISTRY}
-    # First build mender's repo Docker image
```

Comment:
Where is this docker image creation and publish now?

Reply:
Good question! I will rephrase the commit message to explicitly state this.

The thing is that this other image was not in use anymore since https://github.com/mendersoftware/integration/commit/f747c663d1da44a11dc58dfb34ba6e597db6000d so I just removed it.